src/generateNum.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function getNumbers() {
+  const nums = [];
+
+  while (nums.length < 4) {
```

Comment:
not critical but will be good move `4` to the constants and reuse in different places